src/components/likedVideos/LikedVideos.jsx

```diff
@@ -0,0 +1,44 @@
+import { CommonVideoCard } from "components";
+import { useData } from "contexts";
+import { useNavigate } from "react-router-dom";
+
+function LikedVideos() {
+  const { state } = useData();
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <div className="flex-column">
+        <div className="flex-column-start">
+          <div className="large-font-size">Liked Videos</div>
+          <div>{`${state.likes.length} video(s)`}</div>
+        </div>
+
+        <div className="flex-row-center">
+          <div className="videos-container">
+            {state.likes.map((video) => (
+              <CommonVideoCard
+                video={video}
+                playlistCategory="likedVideo"
+                key={video._id}
+              ></CommonVideoCard>
+            ))}
+          </div>
+        </div>
+      </div>
+      {state.likes.length === 0 && (
+        <div className="flex-column-center margin-container no-playlist-container">
+          <div>No liked videos.</div>
+          <button
+            className="btn btn-primary"
+            onClick={() => navigate("/videos")}
```

Comment:
#### Here is a suggestion
I think you can use `Link` here. Whenever possible use the link. Also, router docs have mentioned that you should have a good reason to use navigate. 

Why? 
Here using the button you are linking to a different page, is not the best accessibility practice. Linking to the different pages require a `a` tag.